hack/test-cmd.sh

```diff
@@ -238,6 +238,9 @@ fi
 PATH="${KUBE_OUTPUT_HOSTBIN}":$PATH
 
 kube::log::status "Checking kubectl version"
+
+# Note that this fails on a dirty commit.  To avoid that from happening, .gitignore
```

Comment:
it shouldn't fail if both the client and server were both built at a dirty commit, I think. maybe if one was built before the file was generated? seems odd though.